util/file_util.py

```diff
@@ -629,6 +643,9 @@ def file_write_csv_dict(py_dict: dict,
       is used as the key's column name. If no columns are specified for values,
       column names are picked from each entry's value if the value is a dict.
       Else the value is written as column name 'value'.
+    key_column_name: name of the column used as key.
```

Comment:
For backward compatibility, should the default value of key_column_name not be None instead of 'key' . Previous implementation picked first column as key column. Same behavior would be preserved using None as key_column_name.

That being said, will it be more intuitive to use None to skip key and something else (Enum/constant) to indicate first column as key?